pkgs/development/python-modules/ament-lint-jazzy/default.nix

```
@@ -0,0 +1,32 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  # build-system
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "ament-lint-jazzy";
```

```suggestion
  pname = "ament-lint";
```
according to https://github.com/ros2-gbp/ament_lint-release/blob/release/jazzy/ament_lint/0.17.3-1/setup.py#L4

---

that's not a good idea, because `ament_lint` is ros distribution specific. the jazzy version wouldn't work with humble ros for example. Iwould like to stay with `ament-lint-jazzy`